sources/packages/web/src/components/institutions/confirmation-of-enrollment/information.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <div v-if="showInfo">
+    <div class="bg-white mt-10 mb-4" :class="containerClass">
+      <p>
+        <v-icon :color="iconColor">mdi-information </v-icon
+        ><span class="pl-2 font-weight-bold" :class="infoHeaderClass">
+          {{ infoHeader }}
+        </span>
+      </p>
+      <span class="mt-4">
+        {{ content }}
+      </span>
+    </div>
+  </div>
+</template>
+<script lang="ts">
+import { onMounted, ref, watch } from "vue";
+import { COEStatus, ApplicationDetailsForCOEDTO } from "@/types";
+
+export default {
+  props: {
+    data: {
+      type: Object,
+      default: {} as ApplicationDetailsForCOEDTO,
+    },
+  },
+  setup(props: any) {
+    const content = ref("");
+    const containerClass = ref("");
+    const infoHeader = ref("");
+    const infoHeaderClass = ref("");
+    const iconColor = ref("");
+    const showInfo = ref(false);
+    const prepareContent = () => {
+      showInfo.value = false;
+      if (
```

Comment:
Just as a recommendation, it could be simplified to avoid repeating "_props.data.applicationCOEStatus_" as below:
```
if([COEStatus.submitted, COEStatus.completed].includes(props.data.applicationCOEStatus)){
    // Do seomthing
}
```